common/djangoapps/entitlements/rest_api/v1/filters.py

```diff
@@ -40,7 +40,8 @@ class CourseEntitlementFilter(filters.FilterSet):
 
     uuid = UUIDListFilter()
     user = filters.CharFilter(field_name='user__username')
+    course_uuid = UUIDListFilter(field_name='course_uuid')
```

Comment:
Will query params character length be an issue if there is a large courseUUID list?

Reply:
potentially, yes